server.js

```diff
@@ -32,24 +31,24 @@ if (isSSR) {
         '/de-de/changelog': '/changelog',
         '/zh-cn/changelog': '/changelog',
         '/changelog':
-          'https://raw.githubusercontent.com/docsifyjs/docsify/master/CHANGELOG'
-      }
+          'https://raw.githubusercontent.com/docsifyjs/docsify/master/CHANGELOG',
+      },
     },
-    path: './'
-  })
+    // path: './', // not used for anything?
```

Comment:
Unused?

Reply:
Actually, I found out because I add `// @ts-check` at the top of any JS file and have TypeScript do live type checking in VS Code, and it complained that the option was unused (which appears to be the case).